apps/electron/src/pcap/packet-capture.ts

```diff
@@ -157,7 +158,11 @@ export class PacketCapture {
       options.pid = this.options.pid;
     }
 
-    this.captureInterface = new CaptureInterface(isDev ? options : { ...options, exePath: PacketCapture.MACHINA_EXE_PATH });
+    this.captureInterface = new CaptureInterface(isDev ? options : { 
+      ...options, 
+      exePath: PacketCapture.MACHINA_EXE_PATH,
+      localOpcodesPath: PacketCapture.LOCAL_OPCODES_PATH,
+    });
```

Comment:
You're making it use local opcodes if the env is not dev, which means local opcodes in production. That's not the way it's designed, loading from network has to stay the default behavior, also I'm not sure forcing local opcodes in dev env is the way to go, not everyone starts local dev env to test opcodes ^^

Reply:
The local opcode was designed to be placed manually, which means the behaviour is enforced, not acting as a fallback, and only developers who want to test the opcode should touch the file. If the file not exists, it would fallback to network requests.

But I do agree that it should be placed in dev and have an environment variable which controls this behaviour. Would work on this tomorrow.